src/app.py

```diff
@@ -38,6 +38,7 @@ class GlobalState:
     transitions: Dict = None,
     tick_number: int = 0
```

Comment:
Would the game work if we changed it to `-1` and line 201 to `tick_number < 0`?
That would be more consistent with the frontend where we start with tick number equal to -1
In reference to: https://github.com/group-project-gut/lynx-scene-host-python/pull/17/files#r1268693336

Reply:
Nope, still this bug appears with this change. So it's must be connected to frontend. When we get first map, then our tick_number on front is 0. When we press populate, then it changes to -1, that's why one window `get` the map normally and other not.